### PR TITLE
[FEATURE] Link to and display validator report history

### PR DESCRIPTION
**Pull request by stevenzeiler**
Simply links to existing validator "show" page.

### CODE REVIEW

**Comment by stevenzeiler** [comment]
+r

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
Does this break "Copy to Clipboard", or was it not previously working?

**Comment by stevenzeiler** [comment]
"Copy to Clipboard" was removed from the UI a while back, this commit sweeps up the dangling references.

**Comment by wilsonianb** [comment]
I'm seeing "Copy to Clipboard" when running the webapp locally.

**Comment by wilsonianb** [comment]
Ah, this pull request should be for `staging`

**Comment by stevenzeiler** [comment]
Let's change the default branch to be staging.

File: scripts/controllers/validator.js [hunk 1]
```diff
@@ -28,8 +44,8 @@ angular.module('validatorsApp')
       "n9MG8aiQxrupaCnkvTdLeEN6XGsedSdLd8NnVE9RgfaanPvrspL7": "bitstamp",
       "n9MLVGTjHcKaUComy5ogfd1dZjzuQhqrTMNuefQZmcUPzCj21tX3": "wisepass",
       "n9Mb8ZiVL6bepu2QbSTbPMNVJp4TUj2cRq4qY7A9EMyAdRaKiCnD": "the rock trading"
-    }
+    };
+    */
```
Comment (wilsonianb) [inline scripts/controllers/validator.js]: To comment or not to comment

**Comment by stevenzeiler** [comment]
:+1:

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**